src/AutocompleteBase.js

```diff
@@ -98,7 +98,7 @@ AutocompleteBase.STATE = {
 	 * @type {function}
 	 * @default Identity function.
```

Comment:
Identify functions are functions that return the same value they receive, which is not the case for `listId`. Also, it's not of type `function`, it actually receives a `string`. Can you fix these issues? It's also important to change the description for `listId` since it's not the component's identification, just the id used by the list's main element. Also, `compose` has no `u` letter :)